README.rst

```diff
@@ -321,6 +326,7 @@ Here is an end-to-end example:
     transformer = mxnet_estimator.transformer(1, 'local', assemble_with='Line', max_payload=1)
     transformer.transform('s3://my/transform/data, content_type='text/csv', split_type='Line')
     transformer.wait()
+    transformer.delete_model()
```

Comment:
Can we put a small comment above similar to the one above in regards to delete_endpoint and add a newline.

'# Deletes the SageMaker model'